README.md

```diff
@@ -26,17 +26,23 @@ Currently supported and tested linters:
 ## Requirements
 
 CMake is used as the build system wrapped with my [mz-cmaketools](https://emzeat.de/mz-cmaketools) for convenience.
+These get pulled as git submodule so make sure to run `git submodule update --init` before trying to build.
```

Comment:
Did you consider including the step as part of the build steps below? I find that people tend to miss instructions when instructions are split into multiple spots to find the instructions, often because the instructions are expected to be a single series of instructions.